wptrunner/wptrunner.py

```diff
@@ -148,6 +148,9 @@ def run_tests(config, test_paths, product, **kwargs):
 
         kwargs["pause_after_test"] = get_pause_after_test(test_loader, **kwargs)
 
+        env_options["host"] = kwargs.get("host") or env_options["host"]
+        env_options["external_host"] = kwargs.get("external_host") or env_options["external_host"]
```

Comment:
This could be kwargs.get("host", env_options["host"]). If kwargs.get("host") returns a falsy value (false, 0, "", &c.) that will select env_options["host"], but using the dict.get(key, fallback) method will perform a proper None check.